src/components/Button/CozyHome.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { translate } from 'cozy-ui/react/I18n'
+import { ButtonLink } from 'cozy-ui/react'
+
+const CozyHome = ({ t, from }) => (
+  <ButtonLink
+    label={t('Share.create-cozy')}
+    href={`https://cozy.io/try-it?from=${encodeURIComponent(from)}`}
+  />
+)
+
+CozyHome.propTypes = {
+  t: PropTypes.func.isRequired,
+  from: PropTypes.string
+}
+
+CozyHome.defaultProps = {
+  from: ''
```

Comment:
If we omit `from`, then the link will still contain the query parameter `from`. 

```js
const url = `https://cozy.io/try-it${from ? `?from=${encodeURIComponent(from)}` : ''}`
```

or use the [URL API](https://developer.mozilla.org/en-US/docs/Web/API/URL).